cloud-kotlin-extensions/build.gradle

```diff
@@ -0,0 +1,44 @@
+import org.jetbrains.dokka.gradle.DokkaTask
+
+plugins {
+    id 'org.jetbrains.kotlin.jvm' version '1.4.21'
+    id 'org.jetbrains.dokka' version '1.4.20'
+}
+
+kotlin {
+    explicitApi()
+}
+
+compileKotlin {
+    kotlinOptions {
+        jvmTarget = '1.8'
+    }
+}
+
+compileTestKotlin {
+    kotlinOptions {
+        jvmTarget = "1.8"
+    }
+}
+
+dependencies {
+    api project(':cloud-core')
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8"
```

Comment:
Don't expect Kotlin versions; let the one who pulls it in do this. `compileOnlyApi` is ideal here.